cds/modules/migrator/records.py

```diff
@@ -778,10 +777,6 @@ def _create_missing_subformats(cls, record, deposit):
         max_width = int(master['tags']['width'])
         max_height = int(master['tags']['height'])
 
-        if ratio not in current_app.config['CDS_SORENSON_PRESETS']:
-            ratio = get_closest_aspect_ratio(max_height, max_width)
```

Comment:
I think you still need this line here, if the aspect ratio is not the list the next line, `preset = current_app.config['CDS_SORENSON_PRESETS'][ratio]`, will raise a `KeyError`

Reply:
So, it cannot be deleted? @jbenito3

Reply:
Yep you have to keep it if you don't want to do a big refactoring :)

Reply:
Then I keep it and just bump sorenson.